tests/models/compute/security_group_tests.rb

```diff
@@ -35,6 +35,18 @@
       @group.ip_permissions.empty?
     end
 
+    test("authorize access at a port range (egress rule)") do
+      @group.authorize_port_range(5000..6000, :direction => 'egress')
+      @group.reload
+      @group.ip_permissions_egress.size == 1
```

Comment:
in reality, there are two rules here

```
[2] pry(#<Shindo::Tests>)> @group.ip_permissions_egress
=> [{"groups"=>[], "ipRanges"=>[{"cidrIp"=>"0.0.0.0/0"}], "ipProtocol"=>"tcp", "fromPort"=>5000, "toPort"=>6000}, {"groups"=>[], "ipRanges"=>[{"cidrIp"=>"0.0.0.0/0"}], "ipProtocol"=>"-1"}]
```

I would recommend changing the assertion to verify that the rule you created is present, rather then asserting absolute size.